packages/patternfly-4/react-integration/README.md

```diff
@@ -26,8 +26,7 @@ Once you’ve completed development of a new component, you need to perform a bu
 
 ## Running the demo app
 
-Running the demo app
-Run the following command from the root directory to launch the demo app:
+Run the following command from the `packages/patternfly-4/react-integration/demo-app-ts` directory to launch the demo app:
```

Comment:
This shouldn't be necessary - what is happening when you run it from the root?  There is a start:demo-app entry point in the root package.json file.